analytics_data_api/v0/serializers.py

```diff
@@ -75,6 +75,43 @@ class Meta(object):
         )
 
 
+class ConsolidatedAnswerDistributionSerializer(ProblemResponseAnswerDistributionSerializer):
+    """
+    Serializer for consolidated answer distributions.
+    """
+
+    consolidated_variant = serializers.BooleanField()
+
+    class Meta(ProblemResponseAnswerDistributionSerializer.Meta):
+        fields = (
```

Comment:
you could do `fields = ProblemResponseAnswerDistributionSerializer.Meta.fields + ('consolidated_variant',)` -- maybe that's nicer?